members/P000612.yaml

```diff
@@ -264,4 +264,4 @@ contact_form:
     headers:
       status: 200
     body:
-      contains: "We will contact you"
+      contains: "We will contact you if we need any additional information."
```

Comment:
Adding in the whole string would make it less likely to match, right?

Reply:
Wait, how so? I could see it taking a little bit more to find the string, but why not match?

Reply:
If we're getting errors saying the text does not meet the success criteria, why would our change be making the expected success criteria more specific?

Reply:
Good point. For this change, this time I copy-pasted the exact string from the page. Which I guess is not that much different from writing it out on the yaml, but at least I know that issues with spelling, or maybe extra spaces that I didn't catch, are hopefully not going to be a problem. Hence why I added the whole string rather that just the first part as it originally was. 

I can certainly ignore this change if we think it would really not do anything. I don't currently see any more errors from this yaml, which makes me think the real issue was something else on that form acting up on that form.

Reply:
your changes are fine as they are now!

Reply:
graciasssss!